site/content/docs/multistep-checks/degraded-state.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Degraded state and soft assertions
```

Comment:
`and soft assertions` sounds like documentation about it, instead is more about the use of it for degraded states, wdyt?

```suggestion
title: Degraded state with soft assertions
```